src/Internal/Internal.purs

```diff
@@ -15,6 +15,8 @@ import Record.Builder (Builder)
 import Record.Builder as Builder
 import Type.Data.RowList (RLProxy(..))
 
+data FormProxy (form :: (Type -> Type -> Type -> Type) -> Type) = FormProxy
```

Comment:
Would you mind explaining a little about what this is meant to do? I see its use in `mkLensesFromRow`, but would it have been impossible to implement that without this custom proxy? And are there other parts of the code base that could benefit from its use?